apps-api/backoffice/src/modules/admin/hashtag/repository.ts

```diff
@@ -66,8 +90,8 @@ export class AdminHashtagRepository {
       this.prismaService.hashTag.update({
         where: { id: hashtagId },
         data: {
-          name: data.name,
-          status: data.status,
+          name: data.name ?? Prisma.skip,
+          status: data.status ?? Prisma.skip,
```

Comment:
Can we leave it for now ? I have concerns that it might break other parts in the system so I suggest that we should open another PR for utilizing `Prisma.skip`
```suggestion
          name: data.name,
          status: data.status,
```